readme.md

```diff
@@ -44,6 +44,12 @@ Returns an array of objects in the format `{ pattern: string, opts: Object }`, w
 
 Note that you should avoid running the same tasks multiple times as they contain a file system cache. Instead, run this method each time to ensure file system changes are taken into consideration.
 
+### globby.hasMagic(pattern, [options]);
```

Comment:
Yes that would be much better.

Wasn't even sure if documenting it, was needed at all. Since the goal `Extends glob with support for multiple patterns and exposes a Promise API` already implied it.